packages/demo-wallet-shared/methods/sep24/pollDepositUntilComplete.ts

```diff
@@ -54,7 +54,8 @@ export const pollDepositUntilComplete = async ({
         });
       }
       log.instruction({
-        title: `Transaction \`${transactionId}\` is in \`${transactionJson.transaction.status}\` status`,
+        title: `Transaction \`${transactionId}\` is in \`${transactionJson.transaction.status}\` status. \n
+        Transaction body: \`\`\`${JSON.stringify(transactionJson.transaction)}\`\`\``,
```

Comment:
I think we want to use `log.response` for the transaction body because it has better styling.